csp-mixin/signals/azurefrontdoor.libsonnet

```
@@ -73,8 +99,14 @@ function(this)
         unit: 'percent',
         sources: {
           azuremonitor: {
-            expr: 'avg by (job, resourceGroup, subscriptionName, dimensionEndpoint) (azure_microsoft_cdn_profiles_percentage4xx_average_percent{%(queriesSelector)s}) + avg by (job, resourceGroup, subscriptionName, dimensionEndpoint) (azure_microsoft_cdn_profiles_percentage5xx_average_percent{%(queriesSelector)s})',
-            legendCustomTemplate: '{{dimensionEndpoint}}',
+            expr: 'avg by (job, resourceGroup, subscriptionName, dimensionEndpoint) (azure_microsoft_cdn_profiles_percentage4xx_average_percent{dimensionEndpoint!="",%(queriesSelector)s}) + avg by (job, resourceGroup, subscriptionName, dimensionEndpoint) (azure_microsoft_cdn_profiles_percentage5xx_average_percent{dimensionEndpoint!="",%(queriesSelector)s})',
+            legendCustomTemplate: '{{dimensionEndpoint}} {{dimension_Endpoint}}',
+            aggKeepLabels: ['dimensionEndpoint'],
+          },
+          azuremonitor_agentless: {
+            expr: 'avg by (job, resourceGroup, subscriptionName, dimension_Endpoint) (azure_microsoft_cdn_profiles_percentage4xx_average_percent{dimension_Endpoint!="",%(queriesSelector)s}) + avg by (job, resourceGroup, subscriptionName, dimension_Endpoint) (azure_microsoft_cdn_profiles_percentage5xx_average_percent{dimension_Endpoint!="",%(queriesSelector)s})',
+            legendCustomTemplate: '{{dimensionEndpoint}} {{dimension_Endpoint}}',
```

```suggestion
            legendCustomTemplate: '{{dimensionEndpoint}}{{dimension_Endpoint}}',
```
As you are not expecting both, you might remove trailing whitespace?